src/layers/FCLayer.cpp

```diff
@@ -1 +1,19 @@
 #include "layers/FCLayer.hpp"
+
+size_t Shape::get_index(const std::vector<size_t>& coords) const {
+  size_t res = 0;
+  size_t mulbuf;
```

Comment:
Why it needs to be defined here?

Reply:
To separate complex non-template function from the header file

Reply:
I meant 'mulbuf' outside of for loop

Reply:
Can also place it somewhere under the class definition

Reply:
It's about mulbuf variable

Reply:
It's moved now